django/contrib/gis/db/backends/oracle/operations.py

```diff
@@ -203,10 +203,10 @@ def get_distance(self, f, value, lookup_type):
 
         return [dist_param]
 
-    def get_geom_placeholder(self, f, value, compiler):
+    def get_geom_placeholder_sql(self, f, value, compiler):
         if value is None:
-            return "NULL"
-        return super().get_geom_placeholder(f, value, compiler)
+            return "NULL", ()
```

Comment:
I wonder if this is simply something we should add to he base implementation.

I suggest we do so on Oracle because it uses to have issues with `%s, (None,)` to represent `NULL`.